src/data_collectors.py

```diff
@@ -0,0 +1,71 @@
+import abc
+from concurrent.futures import Future
+from functools import partial
+import requests
+import re
+
+from constants import TIMEOUT_RSS_REQUEST
+from helper import get_request_with_timeout
+
+class BaseDataCollector(object): 
+    def __init__(self):
+        pass
+    
+    @abc.abstractmethod
+    def get_data_collection_futures(self, executor):
+        pass
+
+
+class RssDataCollector(BaseDataCollector): 
```

Comment:
add doc string
maybe with example what an instance of RssDataCollector and its properties would look like

Reply:
also add doc strings to classes function and their respective params